src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooBuildPlanService.java

```diff
@@ -503,6 +504,26 @@
         }
     }
 
+    private static Path getScriptPattern(ProgrammingLanguage programmingLanguage, Optional<Path> subDirectory, boolean sequentialBuildRuns, boolean getScaTasks) {
```

Comment:
Why do you name it pattern, if it is a path? Can you explain how this path relates to a pattern?

Reply:
I added a Javadoc that explains what kind of pattern is returned.